DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -54,10 +54,9 @@ FROM Applications
                   INNER JOIN Sections ON Sections.ApplicationID = Applications.ApplicationID
                   INNER JOIN Centres ON Customisations.CentreID = Centres.CentreID
                   LEFT JOIN Tutorials ON Sections.SectionID = Tutorials.SectionID
-                  LEFT JOIN Progress ON Customisations.CustomisationID = Progress.CustomisationID AND Progress.CandidateID = @candidateId
+                  LEFT JOIN Progress ON Customisations.CustomisationID = Progress.CustomisationID AND Progress.CandidateID = @candidateId AND Progress.RemovedDate IS NULL AND Progress.SystemRefreshed = 0

```

Comment:
Probably worth adding a test for this specific scenario, e.g. getting the course content when there are progress entries with removed date or system refreshed set 🙂